.circleci/cluster/manager1_config.yaml

```diff
@@ -26,7 +26,7 @@ postgresql_server:
 
 postgresql_client:
   ssl_enabled: true
-  postgres_password: 'somesecretvalue123!'
+  server_password: 'somesecretvalue123!'
```

Comment:
Why are we changing the name of this value?

Reply:
Since it won't always be the `postgres` user's password, it didn't make sense to keep this name.
Specifically when creating a PG DB in Azure, we don't get the `postgres` or any superuser for that matter.